packaging/examples/metrics/grafana-dashboards/strimzi-kafka-mirror-maker-2.json

```diff
@@ -20,19 +20,31 @@
       "type": "panel",
       "id": "graph",
       "name": "Graph",
-      "version": "5.0.0"
+      "version": ""
```

Comment:
Can you explain the different changes outside of the new panels? Or try to minimize them if they are not intentional?

Reply:
So I just imported the dashboard from main branch, added the 3 new panels and exported from the sharing menu (external export). I also used the same Grafana version that is on main (7.3.7). I also tried to import it back successfully. Is this procedure correct?

Reply:
Well, TBH when I do some changes, given I have no clue abotu the meanings of the feilds which it decides to change automatically I usually manually revert them back. That gives it a good changce that I do not break anything with my PR.